docs/source/conf.py

```diff
@@ -88,3 +88,8 @@
 # especially regarding anchors that do not redirect properly.
 
 linkcheck_ignore = ["https://www.outreachy.org/docs/community/#", "(.*)?README.md#"]
+
+# -- Custom scripts ----------------------------------------------------------
+import subprocess  # noqa: E402
+
+subprocess.run(["python", "../../helper-scripts/get-repo-issues.py"])
```

Comment:
sys.executable is often a good idea to avoid path issues:

```suggestion
subprocess.run([sys.executable, "../../helper-scripts/get-repo-issues.py"])
```